validate.go

```diff
@@ -106,6 +107,29 @@ func checkMounts(mounts []rspec.Mount, rootfs string) {
 	}
 }
 
+func checkHooks(process rspec.Hooks, rootfs string) {
```

Comment:
On Thu, Apr 28, 2016 at 07:54:26PM -0700, Ma Shimiao wrote:
> `hooks are in the host mount namespace` I'm not sure what it means.
> Do you mean hooks is not in OCI bundle?

The may be in the bundle, but if they are, the path will be
/path/to/bundle/and/hook, not /and/hook.  Specs are [1]:

  “Hook paths are absolute and are executed from the host's
  filesystem.”

Although I'd be happier if the wording used “runtime namespace” [2] so
it was explicit about other namespaces as well (e.g. hooks are also in
the host network namepace, the host ipc namespace, …).

[1]: https://github.com/opencontainers/runtime-spec/blob/v0.5.0/config.md#hooks
[2]: https://github.com/opencontainers/runtime-spec/blob/v0.5.0/glossary.md#runtime-namespace